CHANGELOG.md

```diff
@@ -13,6 +13,9 @@
 - Fixed - `Autopopulate.populate` excludes `reserved` jobs in addition to `ignore` and `error` jobs 
 - Fixed - Issue [#1159]((https://github.com/datajoint/datajoint-python/pull/1159)  (cascading delete)  - PR [#1160](https://github.com/datajoint/datajoint-python/pull/1160)
 
+### 0.14.2 -- July 29, 2024
```

Comment:
The changes above have already been implemented, so your change will have to be rolled into the next release.

Reply:
Hi @dimitri-yatsenko 

> The changes above have already been implemented, so your change will have to be rolled into the next release.

Could you clarify what you meant by "these changes have already been implemented"? Where was this updated/implemented? I looked and didn't see any changes elsewhere according to the public branches of this repo nor in the public forks of this repository, though I may have missed it. 

Thanks in advance

Reply:
The fixes listed in the CHANGELOG are queued for the next release. This issue will be added to that list rather than becoming its own release.

Reply:
Ahh okay. I can fix that. Thanks for letting me know